Packs/CommonTypes/ReleaseNotes/2_8_14.md

```diff
@@ -1,3 +1,7 @@
 
-#### Indicator Types
-**urlRep** - Fixed an issue where the regex caused war room entries to be wrongly parsed.
+#### Incident Fields
+The following are new incident fields:
+- **Alert Type** 
+- **Is Active**
```

Comment:
```suggestion
```

Reply:
Why Is Active? It is a new field in common types